src/cjson/lua_cjson.c

```diff
@@ -49,6 +49,10 @@
 #include "lua_cjson.h"
 #include "strbuf.h"
 #include "fpconv.h"
+// IWYU pragma: no_include <stdbool.h>
+// IWYU pragma: no_include <stdio.h>
+// IWYU pragma: no_include <strings.h>
+// IWYU pragma: no_include "nvim/macros.h"
```

Comment:
```suggestion
```

Reply:
Sorry, but I don't think this is the correct behavior.

Reply:
OK, I have found a builtin way to skip fixing certain files, so we'll be able to not modify the external libraries.